.github/workflows/ci.yaml

```diff
@@ -48,7 +48,12 @@ jobs:
             --tag mgmtapi-3_11 \
             --file Dockerfile-oss \
             --target oss311 \
-            --platform linux/amd64,linux/arm64 .
+            --platform linux/amd64 .
```

Comment:
@johntrimble and/or @tjake I would appreciate your thoughts on this. It seems sometime since I tagged v0.1.14, the multi-platform build step fails with a "context canceled" error, and I haven't been able to figure out why. Splitting up the build into 2 separate builds seems to work. I have tried this locally on a Linux machine with the docker binfmt extras for building ARM targets, and it works as expected without splitting it up, but it seems to fail on GitHub.

Reply:
It's puzzling to me that this would fail, but if releases are still working, I'm not too concerned about having to split them up here. Splitting these like this when doing the release would cause problems (we'd end up with a single arch build and the last one built would win).